pkg/cmd/run.go

```diff
@@ -114,6 +125,8 @@ getenvoy run standard:1.11.1 -- --help
 		"(experimental) location of Envoy's access log server <host|ip:port> (requires bootstrap flag)")
 	cmd.Flags().StringVar(&mode, "mode", "",
 		fmt.Sprintf("(experimental) mode to run Envoy in <%v> (requires bootstrap flag)", strings.Join(envoy.SupportedModes, "|")))
+	cmd.Flags().StringToStringVarP(&templateArgs, "templateArg", "", map[string]string{},
```

Comment:
nit: suffix `*P` in the name `StringToStringVarP` means that the option should have a single letter variant. However, you provide `""`.

In this case it's more appropriate to use `StringToStringVar` (without `*P` suffix)

Reply:
Fixed. Thank you.